region/crateria/east.json

```diff
@@ -1734,6 +1784,137 @@
               ]
             }
           ]
+        },
+        {
+          "from": 15,
+          "to": [
+            {
+              "id": 16,
+              "strats": [
+                {
+                  "name": "G-mode Overload PLMs with Grapple",
+                  "notable": false,
+                  "requires": [
+                    "Grapple",
+                    "Morph"
+                  ]
+                },
+                {
+                  "name": "G-mode Overload PLMs by Bombing Crumble Block with Space Jump",
+                  "notable": false,
+                  "requires": [
+                    "h_canUseMorphBombs",
+                    "SpaceJump",
+                    {"or": [
+                      "h_canCeilingBombJump",
+                      "canBePatient"
+                    ]}
+                  ],
+                  "note": "The crumble block is the leftmost flat ceiling tile."
+                },
+                {
+                  "name": "G-mode Overload PLMs by Bombing Crumble Block",
+                  "notable": false,
+                  "requires": [
+                    "h_canArtificialMorphIBJ",
+                    "h_canArtificialMorphCeilingBombJump"
+                  ],
+                  "note": "Start the IBJ on the second downward tile from the door. The crumble block is the leftmost flat ceiling tile.",
+                  "devNote": "Without canCeilingBombJump, this would take more than 10 minutes."
+                },
+                {
+                  "name": "West Ocean G-mode Overload PLMs by PBing Missile Item Softlock Risk",
```

Comment:
This strat should be removed as it is part of 17 now.  Trying to hit the item from an indirect entry would break.